core/lib/dal/src/transactions_dal.rs

```diff
@@ -972,40 +973,74 @@ impl TransactionsDal<'_, '_> {
     /// These are the transactions that are included to some miniblock,
     /// but not included to L1 batch. The order of the transactions is the same as it was
     /// during the previous execution.
-    pub async fn get_miniblocks_to_reexecute(&mut self) -> Vec<MiniblockReexecuteData> {
-        let transactions_by_miniblock: Vec<(MiniblockNumber, Vec<Transaction>)> = sqlx::query_as!(
+    pub async fn get_miniblocks_to_reexecute(
+        &mut self,
+    ) -> anyhow::Result<Vec<MiniblockExecutionData>> {
+        let transactions = sqlx::query_as!(
             StorageTransaction,
             "SELECT * FROM transactions \
-            WHERE miniblock_number IS NOT NULL AND l1_batch_number IS NULL \
-            ORDER BY miniblock_number, index_in_block",
+                    WHERE miniblock_number IS NOT NULL AND l1_batch_number IS NULL \
+                    ORDER BY miniblock_number, index_in_block",
```

Comment:
Bikeshedding: Is extra indentation a conscious decision? I liked the previously used one, and it's more in line with other multiline SQL queries (including new ones in `BasicWitnessInputProducerDal`).